### PR TITLE
Inheritance Explanation

### PR DESCRIPTION
**Pull request by kashish-goel**
Java Program to enter Employee data (taken as an Example) to present Inheritance. [HACKTOBERFEST 2020]

### CODE REVIEW

File: Programming/Java/Inheritance/Inher.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{
+	public static void main()
+	{
+		Employee e=new Employee();
+		e.name="Samuel";
+		e.age=34;
+		e.phno="767985876";
+		e.adress="4/345,Jopling road";
+		e.salary=40000;
+		e.dispaly();
```
Comment (hryxna) [inline Programming/Java/Inheritance/Inher.java:36]: @kashish-goel it seems you have missed-spelled the word ```adress``` it should be ```address``` 
Also, ```e.dispaly()```  should be ```e.display()```

File: Programming/Java/Inheritance/Inher.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		e.printSalary();
+	
+		Manager m=new Manager();
+		m.name="Joshua";
+		m.age=39;
+		m.phno="631890755";
+		m.adress="5-676,Rajendrapur Rd";
+		m.salary=200000;
+		m.dispaly();
```
Comment (hryxna) [inline Programming/Java/Inheritance/Inher.java:45]: @kashish-goel it seems you have missed-spelled the word ```adress``` it should be ```address``` 
Also, ```e.dispaly()```  should be ```e.display()```
Same as before so it's a humble request please go through your code before raising a PR 😅.

**Review by hryxna — CHANGES_REQUESTED** [review]

**Review by aditya109 — CHANGES_REQUESTED** [review]
Please rectify according to the changes requested.

**Comment by aditya109** [comment]
My bad ! I think I was supposed to merge different PR. I apologize to @kashish-goel and @hryxna !

**Pull request merged**